.github/workflows/docker-check-build.yml

```diff
@@ -0,0 +1,10 @@
+name: docker-check-build
+on:
+  - pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build the Docker image
+        run: docker build . --file Dockerfile --tag s3_file_downloader:$(date +%s)
```

Comment:
No need to tag it, since it's only for check purpose, we don't push it to any Docker Registry so this tag cannot be used

Reply:
Also no need to use `--file` since it's default for Dockerfile name